.github/policies/resourceManagement.yml

```diff
@@ -86,19 +86,6 @@ configuration:
       then:
       - disableAutoMerge
       description: Auto-merge Azure SDK PRs with green builds
-    - if:
-      - payloadType: Issues
-      - titleContains:

```

Comment:
Could the `isAction` condition not have been added here?

Reply:
@jeffhandley I was curious about that too

Reply:
Yes, we could have simply fixed the existing policy config. But the 1ES team recommends breaking the policy up into standalone configs for each functional unit (after migration from FabricBot), as this makes maintenance easier.